packages/grid/stories/grid-cellCustomization.stories.tsx

```diff
@@ -55,8 +56,38 @@ const PercentageCellValue = (props: GridCellValueProps<CurrencyPairRow>) => {
   );
 };
 
-const CellCustomizationTemplate: Story<{}> = () => {
-  const rowData: CurrencyPairRow[] = useMemo(() => {
+const ButtonsCellValue = ({
+  value,
+}: GridCellValueProps<CurrencyPairRow, string>) => {
+  const { getTriggerProps, getTooltipProps } = useTooltip();
+
+  if (!value) return null;
+
+  const openProps = getTooltipProps({
+    title: `Favorite ${value}`,
+    status: "info",
+  });
+
+  return (
+    <FlexLayout
+      className="buttons"
+      align="center"
+      justify="center"
+      direction="row"
+      gap={1}
+    >
+      <Tooltip {...openProps} />
+      <Button {...getTriggerProps<typeof Button>()}>
```

Comment:
The Buttons in this example are in the tab order. When the cell isn't focused it's tab index needs to be -1